### PR TITLE
metrics: Add remote networking packet loss.

### LINKED ISSUE #850: add remote networking packet loss
Measure UDP packet loss for a 64 pkt size in order to obtain acceptable packet loss

### PR DESCRIPTION
**Pull request by GabyCT**
This will measure the UDP packet loss for a 64 pkt in order
to obtain an acceptable packet loss.

Fixes #850

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by devimc** [comment]
lgtm

**Comment by GabyCT** [comment]
@klynnrif could you please check this? thanks

File: metrics/network/remote_network/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Clear Containers remote networking test
 
-- The`remote-networking-iperf3.sh` script measures bandwidth, jitter, latency or parallel bandwidth using
+- The`remote-networking-iperf3.sh` script measures bandwidth, jitter, latency, packet loss or parallel bandwidth using
```
Comment (klynnrif) [inline metrics/network/remote_network/README.md]: Adding series comma: ...measures bandwidth, jitter, latency, packet loss, or parallel bandwidth using

**Comment by GabyCT** [comment]
@klynnrif changes applied

**Pull request merged**